source/_partial/xhr_stubbing_deprecated.md

```diff
@@ -0,0 +1,3 @@
+{% note warning %}
+⚠️ **`cy.server()` and `cy.route()` are deprecated in Cypress 6.0.0**. In a future release, support for `cy.server()` and `cy.route()` will be moved to a plugin. Consider using [`cy.http()`](/api/commands/http.html) instead.
```

Comment:
see comment above. not worth calling out `cy.route()` as a plugin... just straight up suggest using `cy.http()`

Reply:
I'm not sure I agree - unless we are very unsure about future support in a plugin.. The idea of us removing support forever may be intimidating to some users